cdm/core/src/main/java/ucar/nc2/ft2/coverage/CoordAxisHelper.java

```diff
@@ -87,7 +87,7 @@ private int findCoordElementRegular(double coordValue, boolean bounded) {
     double distance = coordValue - axis.getCoordEdge1(0);
     double exactNumSteps = distance / axis.getResolution();
     // int index = (int) Math.round(exactNumSteps); // ties round to +Inf
-    int index = (int) exactNumSteps; // truncate down
+    int index = exactNumSteps > 0 ? (int) exactNumSteps : (int) exactNumSteps - 1; // truncate down
```

Comment:
isn't this the same as using `floor`? I thought casting to int rounds towards zero and floor always truncates down which I think is what you want here.

Reply:
you right, brb